modules/ROOT/edits/download-and-install-service-mesh.adoc

```diff
@@ -0,0 +1,85 @@
+= Downloading and Installing Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, downloading, installing
+
+image::install-service-mesh-breadcrumb.png[]
+
+ 
+== Download Anypoint Service Mesh Client
+
+You install, manage, and troubleshoot your Anypoint Service Mesh using the Anypoint Service Mesh client. 
+
+To download the Anypoint Service Mesh client, type the following command at the command prompt:
+
+`curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > service-mesh && chmod +x service-mesh`
+
+The Anypoint Service Mesh client is now downloaded on your file system.
+
+== Install Anypoint Service Mesh
+
+When you invoke the installer, Istio, Service Catalog, Anypoint Service Mesh Broker, and ClusterIP Service components are installed in your Kubernetes cluster. The installer performs several steps as part of the installation, such as checking for dependencies and performing various configurations. 
+
+=== Install without Proxy
+To install Anypoint Service Mesh, perform the following steps:
+
+. In the location from which you downloaded your Anypoint Service Mesh client, type:
++
+`./service-mesh install`
++
+The `Welcome to Anypoint Service Mesh installer` message is displayed.
+. Enter your client ID that you obtained as part of the prerequisites.
+. Enter your client secret that you obtained as part of the prerequisites.
```

Comment:
```suggestion
. Enter your client secret.
```
Instead of referring to this IN the steps, consider a comment in the task intro about having already met the prereqs, maybe even a link. This repeated wording is awkward.

Reply:
Done! Thanks!